src/components/MaterialUI/DataDisplay/TransferList.js

```diff
@@ -0,0 +1,167 @@
+import React, { useState, useCallback } from "react";
+import { FormattedMessage } from "react-intl";
+import { makeStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import Button from "@material-ui/core/Button";
+import Paper from "@material-ui/core/Paper";
+import classNames from "classnames";
+import { isReactComponent } from "../../../utils/propertyValidator";
+import sharedMessages from "../../../sharedMessages";
+
+const useStyles = makeStyles(theme => ({
+	root: {
+		margin: "auto",
+	},
+
+	title: {
+		color: theme.palette.text.hint,
+		fontSize: theme.spacing(1.2),
+		marginBottom: theme.spacing(0.6),
+	},
+	paper: {
+		overflow: "auto",
+		width: theme.spacing(32.6),
+		height: theme.spacing(48),
+		border: `${theme.spacing(0.1)} solid ${theme.palette.primary.light}`,
```

Comment:
left papaer should have border: `${theme.spacing(0.1)} solid ${theme.palette.grey.borders}`,
right papaer should have border: `${theme.spacing(0.1)} solid ${theme.palette.primary.light}`